src/abv/file_location.py

```diff
@@ -0,0 +1,12 @@
+#pylint: disable=too-few-public-methods
+#pylint: disable=missing-docstring
+
+class FileLocation:
+    #This class allows us to abstract the location for our fetched files
```

Comment:
I think this should be as a docstring, not as a comment.  Like the comment at line 11, not sure whether it's redundant considering it's only one function.  Also, if the docstring is kept, maybe keep away from using possessive pronouns, like us, our, etc.

Reply:
Agreed.